core/spec_initializer.py

```
@@ -0,0 +1,490 @@
+import os
+import json
+from typing import Dict, List, Optional
+from datetime import datetime
+from spec.models import PipelineStage, Manifest, TemplateSpecContent
+from .llm_client import NanoLLMClient
+
+
+class SpecInitializer:
+    """Spec 初始化器"""
+
+    def __init__(self, base_dir: str = None, model: str = "openai/qwen3.5-plus"):
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)